mitodl/ocw-studio
# Added redesigned site header

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #628

#### What's this PR do?
Updates the site header to match the new designs

#### How should this be manually tested?
- On any page outside of the context of a single site (login, site listing, etc), make sure the header looks like it does in the designs, when logged in and logged out
- On any page within the context of a single site (`/sites/<your site>/`, `/sites/<your site>/collaborators/`, `/sites/<your site>/type/page/`, etc), make sure the header looks like it does in the designs
  - This should include the site title and have working preview/publish buttons

#### Any background context you want to provide?
- Designs: [1](https://projects.invisionapp.com/d/main?origin=v7#/console/21537330/456001912/preview) (not on a site detail page), [2](https://projects.invisionapp.com/d/main?origin=v7#/console/21537330/456001917/preview) (on a site detail page)
- We're not implementing a dropdown menu for the user at this time. That was intentionally left off
- I changed the buttons in the header to (a) include the "preview" button, which was left off of the designs, and (b) include the material icons that match the actions that those buttons represent (the designs had a gear icon for "Publish")

#### Screenshots (if appropriate)
![ss 2021-10-08 at 13 15 38 ](https://user-images.githubusercontent.com/14932219/136603838-0cdee88d-8e02-4204-af6c-92e0a3688f92.png)
![ss 2021-10-08 at 13 15 25 ](https://user-images.githubusercontent.com/14932219/136603844-1d1e8850-da56-414c-98db-705e13f80a32.png)
![ss 2021-10-08 at 13 23 54 ](https://user-images.githubusercontent.com/14932219/136607249-ca3f57dd-b65d-42b3-92da-7b38188eccaa.png)
